### DISCUSSION TITLE
Bottomless pit

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4133] [General]
![2022-05-01_12-19-53](https://user-images.githubusercontent.com/104623098/166136341-bee05d0d-1471-4dd6-b3ed-a17dcd013501.png)
Why were the bottomless pits removed?  I downloaded the old version of the bee mod and they look more aesthetically pleasing than deadly goo. How were the bugs caused? If there is a space under the liquid in the editor, it becomes ordinary water. If there is an elevator in it, then in theory, too, what is the point of removal?

**Comment by FrozenDragon0** [comment]
They have been removed because they are going to be replaced by the Chamber exterior system which will make the pits a lot nicer compared to the previous garbage they were.
